meta-test-family.spec

```diff
@@ -1,13 +1,13 @@
 %global framework_name moduleframework
 
-Name:           modularity-testing-framework
+Name:           meta-test-family
```

Comment:
@alexxa Let's discuss it together with other folks. I will create a new issue, so we can merge documentation.